src/query/regu_var.hpp

```diff
@@ -242,6 +243,7 @@ struct regu_ptr_list_node
 inline bool REGU_VARIABLE_IS_FLAGED (const regu_variable_node *regu, int flag);
 inline void REGU_VARIABLE_SET_FLAG (regu_variable_node *regu, int flag);
 inline void REGU_VARIABLE_CLEAR_FLAG (regu_variable_node *regu, int flag);
+inline DB_TYPE REGU_VARIABLE_GET_TYPE (regu_variable_node regu);
```

Comment:
How about using const pointer type or reference type to avoid shallow copy?

1)
```suggestion
inline DB_TYPE REGU_VARIABLE_GET_TYPE (regu_variable_node &regu);
```

2)
```suggestion
inline DB_TYPE REGU_VARIABLE_GET_TYPE (const regu_variable_node *regu);
```

Reply:
good. I fixed it. thanks